modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/dashboard/pages/index.tsx

```diff
@@ -94,6 +97,26 @@ const Dashboard: React.FC<React.HTMLAttributes<HTMLDivElement>> = () => {
 					routeParams={{channelId, groupId}}
 				/>
 			</BasePage.Header>
+			{getMatchedRoute(NAV_ITEMS) === Routes.CONTACTS_INDIVIDUALS && (
+				<BasePage.SubHeader>
+					<div className='d-flex justify-content-end w-100'>
+						<DownloadPDFReport
+							containers={[
+								Containers.CurrentTotalsCard,
+								Containers.EnrichedProfilesCard,
+								Containers.ActiveIndividualsCard,
+								Containers.TopInterestsAsOfYesterday,
+								Containers.DistributionBreakdownCard
+							]}
+							disabled={false}
```

Comment:
```suggestion
							disabled={dataSourceStates.empty}
```

Reply:
And get data from `const dataSourceStates = useDataSource();`

Reply:
done